src/main/kotlin/app/utils/UiHelper.kt

```diff
@@ -0,0 +1,11 @@
+// Copyright 2017 Sourcerer Inc. All Rights Reserved.
+// Author: Anatoly Kislov (anatoly@sourcerer.io)
+
+package app.utils
+
+object UiHelper {
+    fun confirm(message: String): Boolean {
```

Comment:
It's usually a nice idea to require precisely 'y' or 'n', and if the input is something else, it makes sense to repeat the question. An alternative is to have a default (configurable through the call to UIHelper.confirm) that will capitalize the default choice e.g. "y/N"